spec/controllers/admin_stats_controller_spec.rb

```diff
@@ -98,5 +98,32 @@
       end
     end
 
+    describe "depositor counts" do
+      before do
+        GenericFile.new(id: "abc123") do |gf|
+          gf.apply_depositor_metadata user1
```

Comment:
Wrap arg to `#apply_depositor_metadata` in parens here and on lines 108 and 113.